extensions/intellij/intellij_generator_plugin/src/main/java/com/primeholding/rxbloc_generator_plugin/generator/RxTestGeneratorBase.kt

```diff
@@ -172,7 +172,7 @@ abstract class RxTestGeneratorBase(
 
                 if (bloc.stateVariableTypes[index] != "void") {
                     sb.append(" $it != null\n")
-                    sb.append("    ? Stream.value(${it}).shareReplay(maxSize: 1)\n")
```

Comment:
Why we removed the share replay operator ?

Reply:
It is necessary? it seems to compile without it.